bigbite/themer
# [112] - Stop spacing from being cleared

## Description
[112](https://github.com/bigbite/themer/issues/112) - When trying to parse spacing values, if the value was a variable such as ```var(--wp--preset--spacing--20)``` it would just set the value to 0px this PR fixes this issue and also fixes an issue where if you changed the value the value would have no unit at the end. It now defaults this to PX

## Change Log


-  Adds a function to check if a unit exists and if it doesn't it adds PX
-  Returns the original value instead of 0px


## Steps to test
Make sure you have the twenty twenty four theme active
Go to appearance > styles editor
Open the blocks menu and go to the quote block
You should see it has a variable as the value and if you change it a PX value is automatically added.

## Screenshots/Videos

http://bigbite.im/v/kvoKQa

## Checklist:
